src/codeu/chat/client/ClientUser.java

```diff
@@ -171,4 +166,12 @@ public String showUserInfo(String uname) {
   public static void printUser(User user) {
     System.out.println(getUserInfoString(user));
   }
+
+  public int getMessageCount(Uuid userid) {
+      return view.getMessageCount(userid);
+  }
+  
+  public int increaseMessageCount(Uuid userid) {
+	  return controller.updateMessageCount(userid);
```

Comment:
I don't really know if this really matters, but I noticed that our indentions are different. Looks like Google uses spaces and we use tabs.